### PR TITLE
Detect 429 & 413 and append an additional message

### PR DESCRIPTION
**Pull request by vsingh-msys**
### Description

In order to provide better user experince detect know errors 413 (`Request Entity Too Large`) & 413 (`Too Many Requests`) and append additional info. so that the user can easily track and perform adequate actions.

- 413:
```
You most likely hit the erchef request size in Chef Server that defaults to ~2MB. To increase this limit see audit cookbook TROUBLESHOOTING.md OR https://docs.chef.io/config_rb_server.html
```

- 429: 

```
This error typically means the data sent was larger than Automate's limit (4 MB). Run InSpec locally to identify any controls producing large diffs.
```

### Issues Resolved

Fixes https://github.com/chef-cookbooks/audit/issues/384

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

Signed-off-by: Vivek Singh <vivek.singh@msystechnologies.com>

### CODE REVIEW

File: libraries/helper.rb [hunk 1]
```diff
@@ -55,6 +55,10 @@ def handle_http_error_code(code)
       Chef::Log.error 'Auth issue: see audit cookbook TROUBLESHOOTING.md'
     when /404/
       Chef::Log.error 'Object does not exist on remote server.'
+    when /413/
```
Comment (vjeffrey) [inline libraries/helper.rb]: nice!

Comment (vsingh-msys) [reply libraries/helper.rb]: Thanks

**Comment by vsingh-msys** [comment]
Any Update?

**Comment by vjeffrey** [comment]
@vsingh-msys we can merge this pr as soon as the tests go green!

**Comment by alexpop** [comment]
Hey Vivek. Can you please rebase this with master so we can merge? Thank you!

**Comment by vsingh-msys** [comment]
@alexpop rebased it also verified with Automate & Chef server it all working as expected. Thanks

**Comment by alexpop** [comment]
Very good, thank you Vivek!

**Pull request merged**